### PR TITLE
initial content for core

### LINKED ISSUE #261: Add `/core/`
DORA "Core" is a proposed model aimed at easing adoption of our findings by practitioners. We plan a phased rollout onto dora.dev. As a first step, create a new page at `/core/` to describe the concept and present a visualization.

### LINKED ISSUE #262: Link to `/core` from `/research/`
Add a link under the “spider graph” on /research:
“An update to this model is underway: DORA Core.”

### PR DESCRIPTION
**Pull request by davidstanke**
This PR adds a page at `dora.dev/core` to contain the DORA Core model. In its initial state, it won't be linked from the navigation (though there will be a small link from `/research`). We will iteratively augment it until it's ready to replace the content on `/research`

Project planning board: https://github.com/orgs/dora-team/projects/3
Discussion: https://github.com/dora-team/dora.dev/discussions/265

fixes #261
fixes #262

### CODE REVIEW

File: hugo/content/core/_index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
...]({{< relref "/publications" >}})) are continuously released, revealing new insights and dynamics. With each study, some prior findings are reinforced, while others may be called into question. This is a hallmark of good science: any new finding is to be considered suspect—intriguing, but suspect—until it has been validated through replication and application. Meanwhile, practitioners are encouraged to apply the research in their own professional contexts. In such contexts, it can be challenging to keep up with the latest developments from DORA: cultural transformation efforts in a large organization require a steady hand, persistent over stretches of time measured across multiple years. An attempt to update one’s practices to match the pace of the research can lead to counterproductive churn. In these contexts, it’s more practical to rely on “evergreen” artifacts. DORA Core represents a distillation of DORA’s most foundational findings: metrics, capabilities, and outcomes that the research has repeatedly surfaced. It enables teams to focus their improvement efforts even more precisely on what is likely to produce tangible benefits to their organizational goals and quality of life.
```
Comment (nathenharvey) [inline hugo/content/core/_index.md:8]: "Accelerate State of DevOps Reports"?  I can see a good argument both ways.

File: hugo/content/core/_index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<img src="dora-core-model.svg" style="width:100%">
+
+> [Download as PDF](dora-core-model.pdf)
+
+## FAQ
+#### What’s in Core? What isn’t?
+DORA Core encompasses the set of entities that have been perennially studied by DORA and whose interactions are well supported. These entities include capabilities like Continuous Delivery and Generative Culture, metrics like the “Four Key'' metrics, and outcomes like organizational performance. These established ideas contrast with areas of inquiry, or divergent predictive pathways, which have shown promise when we’ve studied them, but aren’t (yet) as well supported as part of DORA’s predictive model. An entity may be considered for Core if: a) it has been researched at least twice; b) practitioners have found it valuable; c) continued exploration is warranted. This rubric is, by design, a subjective one, which draws heavily on the data and analyses, but it is ultimately guided by intuition. So please contribute! Your intuition is valuable; if you feel like something doesn’t accurately reflect the research, please let us know.
```
Comment (nathenharvey) [inline hugo/content/core/_index.md:16]: Should we link to the capabilities when mentioning them? 

i.e., "...like [Continuous Delivery](/devops-capabilities/technical/continuous-delivery/) and [Generative Culture](/devops-capabilities/cultural/generative-organizational-culture/)..."

**Review by nathenharvey — APPROVED** [review]
I'm happy to have these two comments addressed separately and/or after the initial launch and will approve this PR.

**Comment by davidstanke** [comment]
Preview this PR at https://draft-dora-dev--pr264-mnasxkr6.web.app

File: hugo/content/core/_index.md (hunk 2 above)
Comment (davidstanke) [reply hugo/content/core/_index.md]: good call -- fixed

File: hugo/content/core/_index.md (hunk 1 above)
Comment (davidstanke) [reply hugo/content/core/_index.md]: Ah, yes. Accelerate it is!

**Pull request merged**